IMAGE:
  WIDTH: 640
  HEIGHT: 480
  OFFSET: 360
  RECTWIDTH: 320
  RECTHEIGHT: 20
  MARGIN: 12
  KALMANFILTER: false
  # find your ROI parameter.

XYCAR:
  START_SPEED: 7.0
  MAX_SPEED: 0.0
  MIN_SPEED: 0.0
  SPEED_CONTROL_THRESHOLD: 0.0
  ACCELERATION_STEP: 0.00
  DECELERATION_STEP: 0.0
  # find your parameter.

# If you want to use other control methods, Change PID to another control methods.
PID:
  P_GAIN: 0.0
  I_GAIN: 0.0000
  D_GAIN: 0.00
  # find your parameter.

MOVING_AVERAGE_FILTER:
  SAMPLE_SIZE: 30

TOPIC:
  PUB_NAME: /xycar_motor
  SUB_NAME: /usb_cam/image_raw
  PUB_MSG_NAME: /lane_vision
  QUEUE_SIZE: 1

DEBUG: true
